extensions/wikia/Track/Track.php

```diff
@@ -65,6 +65,7 @@ private static function getViewJS ($param=null) {
 		window.beacon_id = result[1];
 	} else {
 		// something went terribly wrong
+		document.write('<img src="http://logs-01.loggly.com/inputs/88a88e56-77c6-49cc-af41-6f44f83fe7fe.gif?message=something%20went%20terribly%20wrong"/>');
```

Comment:
@jcellary - do we have something similar in our infrastructure so we could send simple logs from anywhere? sounds like something useful to have.

I get the point of making it 'quick and dirty' it's just changing something%20went%20terribly%20wrong to wikia_beacon_id is empty 
might save you time later figuring out what went so terribly wrong :)